app/reducers/Session.js

```diff
@@ -112,15 +112,19 @@ export default function session (state = INITIAL_STATE, action) {
       };
 
     case DELETE_SAVED_SESSION_DONE:
-      return omit(state, 'deletingSession');
+      return {
+        ...state,
+        deletingSession: false,
+        capsUUID: null
```

Comment:
i was just doing this to fix an exception that was thrown on delete when capsUUID was still set; the formatted-caps section was having problems with it I think. but we should probably also set caps to null!